packages/mobile/src/app/saga.ts

```diff
@@ -261,7 +261,7 @@ export function* handleDeepLink(action: OpenDeepLink) {
       navigate(Screens.BidaliScreen, { currency: undefined })
     } else if (rawParams.path.startsWith('/cash-in-success')) {
       // Some providers append transaction information to the redirect links so can't check for strict equality
-      const cicoSuccessParam = (rawParams.path.match(/cash-in-success\/(.+)/) || [])[1]
+      const cicoSuccessParam = (rawParams.pathname?.match(/cash-in-success\/(.+)/) || [])[1]
```

Comment:
path includes query params, pathname does not. https://www.npmjs.com/package/url#api